main/parse.c

```diff
@@ -1158,6 +1189,28 @@ static void lazyInitialize (langType language)
 /*
 *   Option parsing
 */
+#define COUNT(D) (sizeof(D)/sizeof(D[0]))
+
+static void lang_def_flag_file_kind_long (const char* const optflag, const char* const param, void* data)
```

Comment:
What is behind us, is behind us.
But can we agree on a naming style and follow it from now on?
At least in this specific case, that code really stands out when compared to the lines above and below!